From f8577e24e837701e843ebfe5758bd137f3e89406 Mon Sep 17 00:00:00 2001
From: Edward Hervey <bilboed@bilboed.com>
Date: Thu, 2 Oct 2014 16:32:53 +0200
Subject: [PATCH] Make lame work with auto* >= 1.12

We just remove the GTK blurb, we're not using it in the gst sdk
---
 configure.in                | 4 ----
 doc/html/Makefile.am        | 2 +-
 doc/man/Makefile.am         | 2 +-
 libmp3lame/i386/Makefile.am | 2 +-
 4 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/configure.in b/configure.in
index 2f8fa66..c8d78a1 100644
--- a/configure.in
+++ b/configure.in
@@ -77,9 +77,6 @@ if test "${GCC}" = "yes"; then
 	AC_MSG_RESULT(${GCC_version})
 fi
 
-dnl more automake stuff
-AM_C_PROTOTYPES
-
 AC_CHECK_HEADER(dmalloc.h)
 if test "${ac_cv_header_dmalloc_h}" = "yes"; then
 	AM_WITH_DMALLOC
@@ -393,7 +390,6 @@ CONFIG_MATH_LIB="${USE_LIBM}"
 
 dnl configure use of features
 
-AM_PATH_GTK(1.2.0, HAVE_GTK="yes", HAVE_GTK="no")
 
 dnl ElectricFence malloc debugging
 AC_MSG_CHECKING(use of ElectricFence malloc debugging)
diff --git a/doc/html/Makefile.am b/doc/html/Makefile.am
index 1a20404..aebb30e 100644
--- a/doc/html/Makefile.am
+++ b/doc/html/Makefile.am
@@ -1,6 +1,6 @@
 ## $Id: Makefile.am,v 1.7 2010/09/30 20:58:40 jaz001 Exp $
 
-AUTOMAKE_OPTIONS = foreign ansi2knr
+AUTOMAKE_OPTIONS = foreign
 
 docdir = $(datadir)/doc
 pkgdocdir = $(docdir)/$(PACKAGE)
diff --git a/doc/man/Makefile.am b/doc/man/Makefile.am
index ecab077..1de89e9 100644
--- a/doc/man/Makefile.am
+++ b/doc/man/Makefile.am
@@ -1,6 +1,6 @@
 ## $Id: Makefile.am,v 1.1 2000/10/22 11:39:44 aleidinger Exp $
 
-AUTOMAKE_OPTIONS = foreign ansi2knr
+AUTOMAKE_OPTIONS = foreign
 
 man_MANS = lame.1
 EXTRA_DIST = ${man_MANS}
diff --git a/libmp3lame/i386/Makefile.am b/libmp3lame/i386/Makefile.am
index e49ad9c..b949a47 100644
--- a/libmp3lame/i386/Makefile.am
+++ b/libmp3lame/i386/Makefile.am
@@ -1,6 +1,6 @@
 ## $Id: Makefile.am,v 1.26 2011/04/04 09:42:34 aleidinger Exp $
 
-AUTOMAKE_OPTIONS = foreign $(top_srcdir)/ansi2knr
+AUTOMAKE_OPTIONS = foreign
 
 DEFS = @DEFS@ @CONFIG_DEFS@
 
-- 
2.1.0

